example/android/app/build.gradle

```diff
@@ -90,13 +90,13 @@ def enableSeparateBuildPerCPUArchitecture = false
 def enableProguardInReleaseBuilds = false
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.0"
+    compileSdkVersion 26
+    buildToolsVersion '26.0.1'
```

Comment:
Does Android developers have any guidelines about quotes?

Reply:
Single quotes are a standard java String. Doublequotes - templattable string. But there is no special guidlines.